website/cue/reference/components/sources/base/aws_s3.cue

```diff
@@ -197,7 +197,7 @@ base: components: sources: aws_s3: configuration: {
 					consumption rate affects the S3 object retrieval rate.
 					"""
 				required: false
-				type: uint: default: 10
+				type: uint: default: 24
```

Comment:
Is this right, or is this the running-system-CPU-cores issue you referenced on Slack?

Reply:
This is the result of the aforementioned CPU thing on Slack, yes. I just finished up for the day before pushing the commit that fixes this. 😅